components/general/SocialMedia/SocialMedia.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "@/styles/SocialMedia.module.css";
+interface SocialMediaProps {
+  isReverse?: boolean;
+}
+const SocialMedia = ({ isReverse = false }: SocialMediaProps) => {
+  const style = isReverse ? styles.reverse : styles.standard;
+  return (
+    <ul className={`${style} ${styles.social_media__ul}`}>
+      <li>
+        <img src="/images/badge-ln.svg" alt="badge" aria-label="badge__ln" />
+      </li>
+      <li>
+        <img src="/images/badge-in.svg" alt="badge" aria-label="badge__in" />
+      </li>
+      <li>
+        <img src="/images/badge-fb.svg" alt="badge" aria-label="badge__fb" />
```

Comment:
These aria-labels should be the ones for a screen reader to read. "badge_ln" is not a proper word for accessibility like aria-label. 

Think about something your machine can read and people can understand the meaning without looking at the screen.

Reply:
I will modify them